03/styles/global.css

```diff
@@ -23,4 +23,77 @@
 
 .v {
     border-color: #baffc9;
-}
\ No newline at end of file
+}
+
+@media screen and (max-width: 599px) {
+    body {
+        display: grid;
+        grid-template-rows: 1fr 1fr 1fr;
+        grid-template-areas: 
+        "p"
+        "t"
+        "v";
+    }
+
+    main {
+        grid-area: t;
+    }
+
+    header {
+        grid-area: p;
+    }
+
+    side {
+        grid-area: v;
+    }
+}
+
+@media screen, (min-width: 600px) and (max-width: 1199px) {
```

Comment:
Tutaj przypominam, że zamiast przecinka ma być `and` (przecinek oznacza `or` tj. lub)